integration/env/common.go

```diff
@@ -0,0 +1,42 @@
+// +build k8srequired
+
+package env
+
+import (
+	"fmt"
+	"os"
+)
+
+const (
+	// EnvVarCircleSHA is the process environment variable representing the
+	// CIRCLE_SHA1 env var.
+	EnvVarCircleSHA = "CIRCLE_SHA1"
+	// EnvVarE2EKubeconfig is the process environment variable representing the
+	// E2E_KUBECONFIG env var.
+	EnvVarE2EKubeconfig = "E2E_KUBECONFIG"
+)
+
+var (
+	circleSHA  string
+	kubeconfig string
+)
+
+func init() {
+	circleSHA = os.Getenv(EnvVarCircleSHA)
+	if circleSHA == "" {
+		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarCircleSHA))
+	}
+
+	kubeconfig = os.Getenv(EnvVarE2EKubeconfig)
+	if kubeconfig == "" {
+		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarE2EKubeconfig))
+	}
+}
+
+func CircleSHA() string {
+	return circleSHA
+}
+
+func KubeConfig() string {
+	return kubeconfig
+}
```

Comment:
The line endings are missing newlines.

Our CI can get pretty picky and sometimes builds may fail. Can you check your editor settings?

Reply:
fixed :)